supplements/fizbans-treasury-of-dragons/spells.xml

```diff
@@ -0,0 +1,198 @@
+			<table>
+                <thead>
+                    <tr><td>d4</td><td>Effect</td></tr>
+                </thead>
+                <tr><td>1</td><td>The smell of apple pie fills the air, and each creature in the cube must succeed on a Wisdom saving throw or become charmed by you until the start of your next turn.</td></tr>
+                <tr><td>2</td><td>Bouquets of flowers appear all around, and each creature in the cube must succeed on a Dexterity saving throw or be blinded until the start of your next turn as the flowers spray water in their faces.</td></tr>
+                <tr><td>3</td><td>Each creature in the cube must succeed on a Wisdom saving throw or begin giggling until the start of your next turn. A giggling creature is incapacitated and uses all its movement to move in a random direction.</td></tr>
+                <tr><td>4</td><td>Drops of molasses hover in the cube, making it difficult terrain until the start of your next turn.</td></tr>
+            </table>
+		</description>
+		<setters>
+			<set name="level">2</set>
+			<set name="school">Illusion</set>
+			<set name="time">1 action</set>
+			<set name="range">60 feet</set>
+			<set name="hasVerbalComponent">false</set>
+			<set name="hasSomaticComponent">true</set>
+			<set name="hasMaterialComponent">true</set>
+			<set name="materialComponent">a piece of crust from an apple pie</set>
+			<set name="duration">Concentration, up to 1 minute</set>
+			<set name="isConcentration">true</set>
+		</setters>
+	</element>
+	<element name="Raulothim's Psychic Lance" type="Spell" source="Fizban's Treasury of Dragons" id="ID_WOTC_FTOD_SPELL_RAULOTHIMS_PSYCHIC_LANCE">
+		<supports>Bard, Sorcerer, Warlock, Wizard, Intelligence, Spell Saving Throw</supports>
+		<description>
+			<p>You unleash a shimmering lance of psychic power from your forehead at a creature that you can see within range. Alternatively, you can utter a creature’s name. If the named target is within range, it becomes the spell’s target even if you can’t see it. If the named target isn’t within range, the lance dissipates without effect.</p>
+			<p class="indent">The target must make an Intelligence saving throw. On a failed save, the target takes 7d6 psychic damage and is incapacitated until the start of your next turn. On a successful save, the creature takes half as much damage and isn’t incapacitated.</p>
+			<p class="indent"><strong><em>At Higher Levels.</em></strong> When you cast this spell using a spell slot of 5th level or higher, the damage increases by 1d6 for each slot level above 4th.</p>
+		</description>
+		<setters>
+			<set name="level">4</set>
+			<set name="school">Enchantment</set>
+			<set name="time">1 action</set>
+			<set name="range">120 feet</set>
+			<set name="hasVerbalComponent">true</set>
+			<set name="hasSomaticComponent">false</set>
+			<set name="hasMaterialComponent">false</set>
+			<set name="duration">Instantaneous</set>
+			<set name="isConcentration">false</set>
```

Comment:
if we're removing the `isRitual` setter when it's false do we want to do the same with the `isConcentration` setter?